acceptance/acceptance_test.go

```diff
@@ -81,16 +84,20 @@ func createSourceFile(ctx context.Context, fname string, content *godog.DocStrin
 	return ctx, os.WriteFile(fpath, []byte(content.Content), 0400)
 }
 
-func createArtifactForFile(ctx context.Context, fname string) (context.Context, error) {
+func createArtifact1(ctx context.Context, path string) (context.Context, error) {
```

Comment:
Nit: `createOneArtifact` ?

Reply:
or `createArtifactForPath` maybe?

Reply:
Removed it, result name is significant so I added it to the previously created scenarios